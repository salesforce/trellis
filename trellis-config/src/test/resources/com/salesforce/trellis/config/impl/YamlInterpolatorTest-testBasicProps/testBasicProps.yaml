properties:
  reason2: this is the reason for the second one
  trellis.rule.reason: this is the default reason
  somewhere: somwehere
  line: just ${an.old} ${sweet.song}
  an: an
  sweet: sweet
  song: song
  sweet.song: ${sweet} ${song}
  an.old: ${an} old

rules:

  - action: DENY
    from:
      - myapp:from1
    to:
      - myapp:to1

  - action: DENY
    from:
      - myapp:from2
    to:
      - myapp:to2
    reason: ${reason2}

  - action: DENY
    from:
      - myapp:from3
    to:
      - myapp:to3
    reason: ${line}
