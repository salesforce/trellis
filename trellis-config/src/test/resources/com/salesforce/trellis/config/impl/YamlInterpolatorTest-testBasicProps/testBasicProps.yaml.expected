rules:

- action: DENY
  from:
  - myapp:from1
  to:
  - myapp:to1
  reason: this is the default reason

- action: DENY
  from:
  - myapp:from2
  to:
  - myapp:to2
  reason: this is the reason for the second one

- action: DENY
  from:
  - myapp:from3
  to:
  - myapp:to3
  reason: just an old sweet song
